web-animations-1/Overview.bs

```diff
@@ -4699,6 +4699,10 @@ interface KeyframeEffect : AnimationEffect {
 
     1.  Set the <a>target element</a> of <var>effect</var> to <var>target</var>.
 
+    1.  If <var>options</var> is a {{KeyframeEffectOptions}} object with a
+        {{KeyframeEffectOptions/pseudoElement}} property, let the <a>target
+        pseudo-selector</a> be the value of that property. Let it be `null` otherwise.
```

Comment:
We need to define the error-handling behavior here (as I just commented on in https://github.com/w3c/csswg-drafts/issues/4586#issuecomment-568360267).

Reply:
If we move it to the same step as `composite` then it gets the same behavior as writing to the setter, which we can then spec (if not already done) as throwing for error cases.

Reply:
Yes, I agree. Although, I think it should still be a separate step, probably before assigning `composite` (so that if _both_ are in error, we throw the error for the `pseudoElement` property instead). But we should use the same sort of prose, i.e. invoking the same error handling as for setting the property.

Reply:
done